Updated the Prerequisites for OTEL docs (#3612)
SumoLogic/sumologic-documentation: docs/integrations/hosts-operating-systems/opentelemetry/linux-opentelemetry.md

- **`sumo.datasource`** - has a fixed value of **linux**.

## Prereqisites
This app is based on the following log files from the Ubuntu Linux machine.

- auth.log
- syslog
- daemon.log
- dpkg.log
- kern.log
- CentOS, Amazon Linux, and Red Hat
- audit/audit.log
- secure
- messages
- yum.log

:::note

> If collection of these log files are already configured (for example, during Linux or Linux - PCI app setup) then no additional log file collection is required. If any of the log files is missing then the missing file collection can be configured in the next step.

Reviewer: ```suggestion
If you've already configured collection of these log files (for example, during Linux or Linux - PCI app setup), then no additional log file collection is required. If any of the log files is missing, you can configure the missing file collection in the next step.
```
